content/en/docs/reference/setup-tools/kubeadm/kubeadm-config.md

```diff
@@ -21,6 +21,9 @@ to print the default configuration and `kubeadm config migrate` to convert your
 files to a newer version. `kubeadm config images list` and `kubeadm config images pull` can be used
 to list and pull the images that kubeadm requires.
 
+In Kubernetes v1.13.0 and later to list/pull kube-dns images instead of the CoreDNS image
+the `--config` method described [here] has to be used.(/docs/reference/setup-tools/kubeadm/kubeadm-init-phase/#cmd-phase-addon)
```

Comment:
`(/docs/reference/setup-tools/kubeadm/kubeadm-init-phase/#cmd-phase-addon)` should follow `[here]`